tests/tests.rs

```diff
@@ -262,6 +262,8 @@ fn links() {
             Test(test::Error, test::ErrorKind);
         }
     }
+
+    {}
```

Comment:
Documented in the commit message: this works around a Rust 1.10 bug (fixed in 1.11 via https://github.com/rust-lang/rust/pull/34436 ), which causes Rust to think that an item-generating macro invoked at the end of a function can only generate expressions instead (and then producing piles of errors when the macro attempts to generate items instead of expressions).